edx_proctoring/templates/onboarding_exam/submitted.html

```diff
@@ -45,6 +45,13 @@
       </button>
     </p>
   {% endif %}
+  <p>
+    {% blocktrans %}
+      If you already have an onboarding profile approved through another course,
```

Comment:
There's something about the way this looks that is a little heavy on the eyes. It's hard for me to see what the call to action is, because the button is now sandwiched between text. What do you think about moving this new copy above "If you have made an error..." but below "Once your profile has been reviewed"?

Reply:
yeah that would make sense, I think this panel is a bit verbose overall but I'd rather over-inform I suppose

Reply:
Alternatively, could we only present certain information if the learner is in another course? This is something that I had outlined in my [discovery work](https://openedx.atlassian.net/wiki/spaces/~579530305/pages/2523103313/MST-673+Discovery+Work) for MST-673. I think it may be easier to read this panel if we're only presenting the relevant info that a learner will need